### PR TITLE
3.3 TokenDrop: Unprotected initialize() function

### PR DESCRIPTION
**Pull request by asselstine**

### CODE REVIEW

**Review by asselstine — COMMENTED** [review]
LGTM

File: contracts/TokenDrop.sol [hunk 1]
```diff
@@ -77,7 +77,7 @@ contract TokenDrop is Initializable {
     /**
      * @notice Initialize TokenDrop Smart Contract
      */
-    function initialize(address _measure, address _asset) external {
+    function initialize(address _measure, address _asset) external initializer {
         measure = IERC20Upgradeable(_measure);
         asset = IERC20Upgradeable(_asset);
```
Comment (asselstine) [inline contracts/TokenDrop.sol:82]: Both these need to be validated!

Comment (kamescg) [reply contracts/TokenDrop.sol:82]: hmmm... apologies i think some of the changes got incorrectly merged when doing the the fix/x.x branches.

will go through each file more closely again.

Comment (kamescg) [reply contracts/TokenDrop.sol:82]: Just want to confirm for validation you mean...

```
require(_measure != address(0), "Pod:invalid-measure-token");
require(_asset != address(0), "Pod:invalid-asset-token");
```

Or validated meaning the `asset` address matches the related TokenFaucet asset?

Comment (asselstine) [reply contracts/TokenDrop.sol:82]: Yes, exactly just making sure they aren't zero.  The faucet isn't visible to this contract so I don't think you could validate that the asset matches the faucet in this location

**Pull request merged**